src/ServiceControl.Transports.ASBS/Helper.cs

```diff
@@ -0,0 +1,40 @@
+﻿namespace ServiceControl.Transports.ASBS
+{
+    using System;
+    using System.Security.Cryptography;
+    using System.Text;
+    using NServiceBus;
+
+    public static class Helper
+    {
+        public static void ConfigureTransport(this TransportExtensions<AzureServiceBusTransport> transport, TransportSettings transportSettings)
+        {
+            transport.ConfigureSanitization();
+
+            transport.Transactions(TransportTransactionMode.SendsAtomicWithReceive);
+            transport.ConnectionString(transportSettings.ConnectionString);
+        }
+
+        public static void ConfigureSanitization(this TransportExtensions<AzureServiceBusTransport> transport)
```

Comment:
We've intentionally moved away from the term "Sanitization" in the new transport. This might be better named as `ConfigureNameShorteners`.

Reply:
I chose this name because of the desired backward compat so I felt having the naming of ASB makes more sense. Happy to change tough